src/Mod/Spreadsheet/Gui/DlgSettings.ui

```diff
@@ -26,6 +26,45 @@
       <string>Import/Export Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
+      <item row="3" column="2">
+       <widget class="Gui::PrefLineEdit" name="formatString">
+        <property name="text">
+            <string  notr="true">%V = %A</string>
+        </property>
+        <property name="placeholderText">
+            <string>%V = %A</string>
```

Comment:
```suggestion
            <string notr="true">%V = %A</string>
```
I'm not sure it's a good idea to have a placeholder text, as when seeing it, user may think he/she set it as a value, while actually an empty string will be stored and feature will be disabled.